src/input/controls.rs

```diff
@@ -117,3 +117,26 @@ pub fn handle_keypress_screen_too_small<B: Backend>(evt: Event, app: &mut App<B>
         _ => (),
     };
 }
+
+pub fn handle_keypress_exiting_mode<B: Backend>(
+    evt: Event,
+    app: &mut App<B>,
+) {
+    match evt {
+        Event::Key(Key::Ctrl('c'))
+        | Event::Key(Key::Char('q'))
+        | Event::Key(Key::Esc)
+        | Event::Key(Key::Backspace)
+        | Event::Key(Key::Char('n')) => {
+            if app.loaded {
```

Comment:
There is an `app.reset_ui_mode()` method that does this. You can use it in both of these conditions.

Reply:
Sorry! I missed the method.
Thanks for the tip!